lld/test/ELF/systemz-pie.s

```diff
@@ -0,0 +1,59 @@
+# REQUIRES: systemz
+# RUN: llvm-mc -filetype=obj -triple=s390x-unknown-linux %s -o %t1.o
+
+## Default is no PIE.
+# RUN: ld.lld %t1.o -o %t
+# RUN: llvm-readobj --file-headers --sections -l --symbols -r %t \
```

Comment:
Suggest that this is added to `basic-*` and switch to `llvm-readelf`

Reply:
Done.